PrivateHospitals/backend/PrivateHospitals/PrivateHospitals.Core/Models/Appointment.cs

```diff
@@ -7,8 +7,9 @@ namespace PrivateHospitals.Core.Models;
 
 public class Appointment
 {
-    [Key]
+    [Key] 
     public int AppointmentId { get; set; }
+    public string? ExternalId { get; set; }
```

Comment:
We agreed we should support search by this fields. Do u want us to fullscan Users table?